lib/reporting/reports/enterprise_fee_summary/enterprise_fees_with_tax_report_by_producer.rb

```diff
@@ -0,0 +1,292 @@
+# frozen_string_literal: true
+
+module Reporting
+  module Reports
+    module EnterpriseFeeSummary
+      class EnterpriseFeesWithTaxReportByProducer < ReportTemplate
```

Comment:
I think we need to add to the Rubocop `_todo` list. There's not easy way of complying to the styles here. It's part of the current report architecture to have those long methods. It would be good to fix that at some point.